src/Microsoft.Azure.WebJobs/TraceLevelAttribute.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-
-namespace Microsoft.Azure.WebJobs
-{
-    /// <summary>
-    /// Attribute used to specify <see cref="TraceLevel"/> for job functions.
-    /// This setting will affect both Console logging as well as Dashboard logging.
-    /// </summary>
-    /// <remarks>
-    /// For very high throughput functions where performance is key you can set the
-    /// level to <see cref="TraceLevel.Off"/> if you don't want any logs (Console or
-    /// Dashboard) to be produced. Dashboard logging is somewhat expensive, so this
```

Comment:
Is this scenario preserved?

Reply:
Doesn't look like it. Why not? It was useful - should be easy to carry forward for ILogger

Reply:
You can use the Filters now, rather than Attributes. You can select which levels you want for each function.